package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cesium-webpack-example",
+  "version": "1.0.0",
+  "description": "The minimal recomended setup for an app using Cesium with Webpack.",
+  "main": "index.js",
+  "keywords": [
+    "cesium",
+    "webpack",
+    "example"
+  ],
+  "author": {
+    "name": "Analytical Graphics, Inc.",
+    "url": "https://www.agi.com"
+  },
+  "homepage": "https://cesiumjs.org",
+  "license": "ISC",
+  "devDependencies": {
+    "cesium": "^1.37.0",
```

Comment:
Up to you, but since I merged your Cesium PR, you can now just use `http://cesium-dev.s3-website-us-east-1.amazonaws.com/cesium/master/cesium-1.38.0.tgz` here and everything will work out of the box.  We can then update it to `1.38` as soon as the release is out.

Reply:
Sounds good.